app/javascript/packages/document-capture/components/review-issues-step.tsx

```diff
@@ -159,7 +159,7 @@ function ReviewIssuesStep({
               showAlternativeProofingOptions={!isFailedResult}
               heading={t('components.troubleshooting_options.ipp_heading')}
               altInPersonCta={t('in_person_proofing.headings.cta_variant')}
-              altInPersonPrompt={t('in_person_proofing.body.cta.prompt_detail_a')}
+              altInPersonPrompt={t('in_person_proofing.body.cta.prompt_detail')}
```

Comment:
We can omit this line entirely if we leave the fallback in `in-person-call-to-action.tsx`.

Reply:
Tying this comment into your previous statement "could we instead leave this file as-is and remove unnecessary props in the components calling this one?"

This is the only place where the input prop matches the fallback text, so this is the only prop that can be removed FYI.